src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/RazorCodeBlockFoldingProvider.cs

```diff
@@ -29,6 +29,7 @@ public override async Task<ImmutableArray<FoldingRange>> GetFoldingRangesAsync(D
                     StartLine = startLine,
                     EndCharacter = endOffset,
                     EndLine = endLine,
+                    CollapsedText = codeBlock.DirectiveDescriptor.Directive
```

Comment:
I'm not sure how I feel about this. This is what makes `@code {` fold into just `code`

Reply:
I think it would be better if it said `@code`, but I like it in general.

Pity collapsing the `@foreach` doesn't do a similar thing

Reply:
I like the `@` in front too. I wonder if we can special case these keywords?

Reply:
![image](https://user-images.githubusercontent.com/475144/172467524-2a776707-ef43-4623-82cd-e78a63d3acc2.png)


Agree this is better. These will always be @code blocks afaict